src/routes/health_check.rs

```diff
@@ -1,5 +1,6 @@
-use axum::{http::StatusCode, response::IntoResponse};
+use axum::{http, response::IntoResponse};
```

Comment:
Switching to `http::StatusCode` because there is another `StatusCode` enum at the base of `reqwest`